README.md

```diff
@@ -160,6 +160,18 @@ optional arguments:
   --params PARAMS       comma separated key value pairs (key1=val1,key2=val2)
   --get-health          Get the health status of the server
   --get-identifier      Identify important information regarding the server and the system
+  --list-output-configuration-sources OUTPUT_ID
+                        Get available configuration sources for a given experiment and output
+  --list-output-configuration-source TYPE_ID
```

Comment:
Is it missing OUTPUT_ID ?

Oh, I see it has to be provided as a separate argument. It's not really clear which arguments must be provided together. But that's the way it is I guess. The user will get a proper error message when missing.

Reply:
Yes, it's not clear at all. That's why I added the examples below to show all options needed per command. The help output (`./tsp_typescript_client --help`) is not good.